### PR TITLE
feat(amount-send): rebrand AmountSend component into Amount

### PR DESCRIPTION
**Pull request by etharner**

### CODE REVIEW

**Review by Liubov-crypto — COMMENTED** [review]
Balances: 
Send ae tokens- pass 
Send Fungible tokens - pass 
Send Tips - pass 
Payments (withdraw/withdraw via scan QR) - pass 
Tips - pass 
Create Invite link - pass 
Top up invite link - pass 
Claim back invite link - pass 

Red border in FF extension and web wallet in FF browser after entering amount (I haven't seen such a red border in other browsers and extensions):
https://user-images.githubusercontent.com/69896204/113871974-18c2d680-97bc-11eb-8756-62a5a71b8c48.mov

AE Send

File: src/popup/router/components/Amount.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="amount">
```
Comment (imollov) [inline src/popup/router/components/Amount.vue:2]: Maybe `AmountInput` can better describe the purpose of the component?

File: src/popup/router/components/Amount.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { mapGetters, mapState } from 'vuex';
+import InputField from './InputField';
+
+export default {
+  components: {
+    InputField,
+  },
+  subscriptions() {
+    return pick(this.$store.state.observables, ['tokenBalance', 'balanceCurrency']);
+  },
```
Comment (davidyuk) [inline src/popup/router/components/Amount.vue:33]:
```suggestion
```
or are they used somewhere here?

File: src/popup/router/components/Amount.vue [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      type="number"
+      v-bind="$attrs"
+      :error="$attrs.error || $attrs.value < 0"
+      :error-message="$attrs.errorMessage || $t('pages.tipPage.amountError')"
+      :placeholder="$t('pages.tipPage.amountPlaceholder')"
+      :label="$attrs.label || $t('pages.tipPage.amountLabel')"
+      @input="$emit('input', $event)"
+    >
+      <template slot="right">
+        <span class="token"> {{ selectedToken ? selectedToken.symbol : $t('ae') }}</span>
```
Comment (davidyuk) [inline src/popup/router/components/Amount.vue:13]:
```suggestion
        <span class="token">{{ selectedToken ? selectedToken.symbol : 'AE' }}</span>
```

File: src/popup/router/components/Amount.vue [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        <span class="amount" data-cy="amount-currency">
+          {{ `(${selectedToken ? formatCurrency(0) : formatCurrency(currencyAmount)})` }}
```
Comment (davidyuk) [inline src/popup/router/components/Amount.vue:15]:
```suggestion
          ({{ formatCurrency(selectedToken ? 0 : currencyAmount) }})
```

File: src/popup/router/components/Amount.vue (hunk 1 above)
Comment (davidyuk) [reply src/popup/router/components/Amount.vue:2]: Is this wrapping div necessary?

**Review by davidyuk — CHANGES_REQUESTED** [review]

**Review by Liubov-crypto — COMMENTED** [review]
Tested, in general LGTM. 
Except that if I enter too big amount, some items will go outside the wallet (extension, web wallet, web wallet in mobile browser):
зображення_viber_2021-04-21_17-52-52
11зображення_viber_2021-04-21_17-52-52

also found this issue, reproducible on develop branch #1028

**Pull request merged**